src/processor/Package.ts

```diff
@@ -228,4 +229,59 @@ export class Package {
       pullAt(sd.rules, rulesToRemove);
     });
   }
+
+  private removeDefaultSlicingRules(): void {
+    [...this.profiles, ...this.extensions].forEach(sd => {
+      const rulesToMaybeRemove: number[] = [];
+      sd.rules.forEach((rule, i, allRules) => {
+        if (
+          rule instanceof ExportableCaretValueRule &&
+          (rule.path.endsWith('extension') || rule.path.endsWith('modifierExtension'))
+        ) {
+          const hasSlicingDiscriminatorTypeRule =
+            rule.caretPath === 'slicing.discriminator[0].type' &&
+            rule.value instanceof FshCode &&
+            rule.value.code === 'value';
+          const hasSlicingDiscriminatorPathRule =
+            rule.caretPath === 'slicing.discriminator[0].path' && rule.value === 'url';
+          const hasSlicingOrderedRule =
+            rule.caretPath === 'slicing.ordered' && rule.value === false;
+          const hasSlicingRulesRule =
+            rule.caretPath === 'slicing.rules' &&
+            rule.value instanceof FshCode &&
+            rule.value.code === 'open';
+          const hasContainsRule = allRules.some(
+            otherRule => otherRule instanceof ExportableContainsRule && otherRule.path === rule.path
+          );
+          const hasOneSlicingDiscriminatorRule = !allRules.some(
+            otherRule =>
+              otherRule instanceof ExportableCaretValueRule &&
+              otherRule.caretPath !== 'slicing.discriminator[0].type' &&
+              otherRule.caretPath !== 'slicing.discriminator[0].path' &&
+              otherRule.caretPath.startsWith('slicing.discriminator[')
+          );
```

Comment:
Won't this return `false` if there is *any* path that has more than one slicing discriminator?  But really we only care if _this_ path has more than one slicing discriminator.  Right?

Reply:
Ah yes. I'll add that check and a test to ensure it works!